modules/silicon/README.md

```diff
@@ -0,0 +1,130 @@
+# RAD Lab Silicon Module
+
+## GCP Products/Services 
+
+* AI Platform Notebooks
+* BigQuery
+* Cloud Storage
+* Virtual Private Cloud (VPC)
+
+## Reference Architechture Diagram
+
+Below Architechture Diagram is the base representation of what will be created as a part of [RAD Lab Launcher](../../radlab-launcher/radlab.py).
+
+![](../../docs/images/V1_DataScience.png)
+
+## IAM Permissions Prerequisites
+
+Ensure that the identity executing this module has the following IAM permissions, **when creating the project** (`create_project` = true): 
+- Parent: `roles/orgpolicy.policyAdmin` (OPTIONAL - Only when setting the Org policy in `modules/[MODULE_NAME]/orgpolicy.tf` as part of RAD Lab module)
+- Parent: `roles/resourcemanager.projectCreator`
+- Project: `roles/storage.objectViewer`
+
+When deploying in an existing project, ensure the identity has the following permissions on the project:
+- `roles/compute.admin`
+- `roles/resourcemanager.projectIamAdmin`
+- `roles/iam.serviceAccountAdmin`
+- `roles/storage.admin`
+- `roles/notebooks.admin`
+
```

Comment:
we also need `serviceAccountUser` in order to deploy container based notebook (with `proxy-mode: service_account`)